config/crd/bases/operator.kyma-project.io_kymas.yaml

```diff
@@ -236,6 +236,21 @@ spec:
                         lookup to be necessary that maybe picks a different ModuleTemplate,
                         which is why we need to reconcile.
                       type: string
+                    customStateCheck:
+                      description: CustomStateCheck for advanced Module State determination
+                      properties:
+                        jsonPath:
+                          description: JsonPath specifies the JSON path to the state
```

Comment:
Please run `make manifests` to reflect the description changes.

Reply:
sorry missed that :)